### DISCUSSION TITLE
widget 'Save Data' doesn't give options described in the documentation

### DISCUSSION THREAD

**Discussion by michous** [biolab/orange3#6084] [Q&A]
Hello, 
First of all, I hope I use the appropriate tracker to report, I hesitate somehow...
According to the Documentation, the widget 'Save Data'  has options to save data with various extensions
Actually I could not access the window with these options such as described in the doc, and the only possibility is pkl extension
A warm thanks for your attention at this ticket
Michel Souweine

**What's wrong?**

**How can we reproduce the problem?** 

**What's your environment?**

- Operating system:
- Orange version:
- How you installed Orange:

**Comment by ajdapretnar** [comment]
Your issue is not described in too much detail, but typically if you only see the pickle option, this means the data you are trying to save is in a format that can only be saved as a pickle. Provide a screenshot of your workflow if you need more information.

I am also converting this to a discussion.

**Reply by michous** [reply]
Thank for your kind answer,
Please see the workflow with its circled save data widget (I hope I dragged it properly)
I added a zone of the table resulting of the hierarchical classification
By the way I use orange version 3.32.0 which a downloaded with [Orange3-3.32.0-Miniconda-x86_64.exe (64 bit)]
![Workflow_for_save_data](https://user-images.githubusercontent.com/9611497/182117242-f47abdc5-8a24-468c-992d-e675fc69c173.png)

**Reply by ajdapretnar** [reply]
Yes, you are trying to save sparse data from the bag of words, which cannot be easily stored in a .csv or such. Thus you will only see the pickle option. You can use Select Columns to remove all bow features and then you will be able to store the data as .csv.

**Reply by michous** [reply]
I tried to get rid of bow features as you kindly advised to do (please see the inclosed workflow) but without having .cvs option available for the Save Data
Nevertheless, on the output data resulting from the Select Columns (please see on the attach workflow), no bow features appear any longer. Didn't I manage properly ? Thanks

![image](https://user-images.githubusercontent.com/9611497/182328375-20e1d902-39b0-4396-9661-8881e2a007d3.png)

**Reply by ajdapretnar** [reply]
It doesn't seem like you properly removed bow features and they are still reported to be on the output.
Also, you'll need those features for clustering. Remove them just before Save Data. And also do not pass the data from the Data table. Data table is just for viewing.

**Comment by michous** [comment]
I Tried to modify my workflow, in accordance with to your last hint, but the csv export option for the Save Data remains unavailable.
Please, Is there any remaining error ?
Thanks
![Workflow_for_save_data_3](https://user-images.githubusercontent.com/9611497/182566959-f94022c5-8c62-4e57-b3cf-d1b97a193016.png)
